docker/src/main/dockerhub-hypersql/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM apache/calcite-avatica:1.11.0
```

Comment:
Needs AVATICA_VERSION in here to make sure that we release a docker container with the appropriate version of Avatica we're releasing. Looks like sure docker/src/test/scripts/verify-dockerhub-dockerfile-version.groovy also needs updates to look at this new directory.